minidump-processor/fuzz/fuzz_targets/walk_stack.rs

```diff
@@ -0,0 +1,67 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use minidump::format::CONTEXT_X86;
+use minidump::{MinidumpContext, MinidumpContextValidity, MinidumpMemory, MinidumpRawContext};
+use minidump::{MinidumpModule, MinidumpModuleList};
+use minidump_processor::walk_stack;
+use minidump_processor::{string_symbol_supplier, CallStack, Symbolizer};
+use std::collections::HashMap;
+use test_assembler::Section;
+
+struct TestFixture {
+    pub raw: CONTEXT_X86,
```

Comment:
Hm. I was going to suggest you could get a little more coverage by making this generic over the various supported CPU contexts, but I guess that's annoying. We should add `From<CONTEXT_X86>` et. al. impls for [`MinidumpRawContext`](https://github.com/luser/rust-minidump/blob/a32419975ba6f190081a148f04a41fc42aa4eb26/minidump/src/context.rs#L22) so that you could make this like:

```rust
struct TestFixture<C: Into<MinidumpRawContext>> {
    pub raw: C,
    // …
}
```

Having said that, I don't know how easy it is to write fuzz targets that test generics like that.